src/CLI/Options.cs

```diff
@@ -29,6 +29,9 @@ public class Options : IOptions
         [Option(Required = false, HelpText = "DelayInSeconds", ResourceType = typeof(Resources.OptionsHelpText))]
         public int DelayInSeconds { get; set; }
 
+        [Option("CustomUIA", Required = false, HelpText = "CustomUia", ResourceType = typeof(Resources.OptionsHelpText))]
```

Comment:
You have extra data here that is confusing the parser
```suggestion
        [Option(Required = false, HelpText = "CustomUia", ResourceType = typeof(Resources.OptionsHelpText))]
```